ex8JQ/phonebook.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style/style.css">
+  <script data-main="js/config" src="js/lib/require.js"></script>
+  <script type="application/javascript">
+    window.onload = function() {
+      require(['config'], function() {
+        require(['jquery'], function($) {
```

Comment:
You should be able to join multiple require statements using more than one dep:
`require(['config', 'jquery'], function(config, $) {`